### PR TITLE
data:OGEmanuel

### PR DESCRIPTION
**Pull request by OGEmanuel**

### CODE REVIEW

File: data/OGEmanuel.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "English major turned **Tech bro**. Love to code, eat, watch football and watch movies",
+  "avatar": "https://github.com/OGEmanuel.png",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/its_GODwithMe",
+      "icon": "FaTwitter"
+    }
```
Comment (Aadarsh805) [inline data/OGEmanuel.json:12]: missing commas after each link

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/OGEmanuel.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      "icon": "FaGithub"
+    }
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/emmanuel-ogunmola-a5007a1a6/",
+      "icon": "FaLinkedin"
+    }
+    {
+      "name": "Gmail",
+      "url": "emanuolumide@gmail.com",
```
Comment (Aadarsh805) [inline data/OGEmanuel.json:25]: make it a mail to link like this `mailto:emanuolumide@gmail.com`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/OGEmanuel.json (hunk 2 above)
Comment (Aadarsh805) [reply data/OGEmanuel.json:25]: also use `FaEnvelope` this for the email icon

**Comment by kumarsonsoff3** [comment]
Closing this one because it seems to be a duplicate to PR #3481

**Pull request closed without merging**